labcodes/labstorybook
# DSYS-22/tags-js

**Link to task:**
https://labcodes.atlassian.net/browse/DSYS-22

**How to test:**
- With the LabSystem running, go to the Component/Button div in the LabStoryBook, then check if all components are rendered as expected.
- Locally, from labstorybook dir, run `npm run test -- -u`, then check if _all_ tests are passing.

**Description of your solution:**
This PR refactor the architecture of `Tags`.  It introduces `AbstractTag` and old `Tag` is now deprecated. `AbstractTag` serve as a "private" component, to be used for building "public"  components of type: `SimpleTag`, `RemovableTag`, `TogglableTag` or `DropdownTag`. `Tag.stories` is refactored accordingly, also including a `TagPlayground`. Tests are implemented in this PR as well.
